### DISCUSSION TITLE
XML + XPath help

### DISCUSSION THREAD

**Discussion by danburd** [FreshRSS/FreshRSS#6250] [Q&A] [answered]
Hello,

I'm decent at creating feeds under the ```HTML + XPath``` setting, but I've never done so with ```XML + XPath``` and I can't seem to get it working. Here is an example feed:

https://www.reuters.com/arc/outboundfeeds/news-sitemap/?outputType=xml&from=100

Here is my best effort, which results in an error:

![image](https://github.com/FreshRSS/FreshRSS/assets/4601223/163b1444-79e0-44eb-b873-0ddd4af425fc)

Thank you very much.

**Comment by Alkarex** [comment]
The XML document is using namespaces. You may try to use a function like [`local-name`](https://developer.mozilla.org/en-US/docs/Web/XPath/Functions/local-name) as such: `*[local-name()='title']`

**Reply by danburd** [reply]
Thank you - that gets me part of the way there. I was able to get the list of items and their URL's, but I couldn't scrape any of the nested items - titles, images, etc.

![image](https://github.com/FreshRSS/FreshRSS/assets/4601223/a586460f-cf6c-44a2-85d2-89b77bd8ff2d)

![image](https://github.com/FreshRSS/FreshRSS/assets/4601223/054f117c-9513-4128-b5f6-1bd399815ee0)

I've tried with and without leading slashes (```//```). Maybe the colon is getting in the way? I assume I'm missing something obvious but I'd be grateful if you could point me in the right direction. Thank you again.

**Reply by Alkarex** [reply]
You still need the `descendant::` part

**Reply by danburd** [reply]
The result was the same with the descendant part, although maybe I'm using it incorrectly.

![image](https://github.com/FreshRSS/FreshRSS/assets/4601223/fbb5b566-f77e-4e50-81db-59727457ee22)

**Reply by Alkarex** [reply, marked as answer]
I believe you must only have the local name part, excluding the namespace prefix, such as `descendant::*[local-name()='title']`

**Reply by danburd** [reply]
Thank you. That was what I needed. I am grateful for your help! This is the setup, for anyone who may be curious:

![image](https://github.com/FreshRSS/FreshRSS/assets/4601223/4c279828-f9d7-4d7f-96bc-55227deb8f64)
